mapzap/models/summary/summary_mapzap_bbls.sql

```diff
@@ -0,0 +1,57 @@
+with project_geometries as (
+    select * from {{ ref('product_mapzap') }}
```

Comment:
I feel like less extra aliases (just referencing source tables as `{{ ref(...) }}` in the queries below) could keep it cleaner as to where data is coming from and how it's being used in the actual meat of the sql file. I take it you prefer this aliasing?

Reply:
That's a good point that you get a clean summary of the sources, I don't know if it quite wins me over though lol. I'm in the middle. Wouldn't a couple of comments listing sources get the job done in fewer lines for that purpose?

I think your last point is good too - if we go this route, just keep the exact names to avoid confusion.

Reply:
I made the "exact name" change in the summary models with https://github.com/NYCPlanning/db-zap-opendata/pull/40/commits/0c4867ca709223baf909268bf0b81e1165b74f71

as I started to do it to the product and intermediate models though, I hit what feels like the main value of an alias: to address tables with a shorter or more meaningful name AND decouple the code from the table name

if a source table name changes, we can either:
- use an alias and change one line at the top of the file
- not use an alias and have to change multiple lines of code and comments

Reply:
Hmm. Alright, that seems fine for now, but definitely this a norm to keep thinking about. We might have made a compromise that leaves neither of us happy lol